src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/repository/ContactsRepository.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity.Contact
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity.ContactType
+
+/**
+ * This repository is read-only and accessed via the view v_all_contacts.
```

Comment:
There is some code in activities we could borrow for a read only repo [here](https://github.com/ministryofjustice/hmpps-activities-management-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/repository/ReadOnlyRepository.kt#L9) meh

Reply:
Can add @Immutable too?